resources/js/Pages/EventPage.vue

```diff
@@ -13,123 +16,101 @@ import {
   StarIcon,
   CalendarIcon,
   UsersIcon,
-  UserIcon,
 } from '@heroicons/vue/24/outline'
 
-const bannerSrc = 'https://picsum.photos/100/100'
-const eventDate = '26 maja 2025 – 12:00'
-const eventTitle = 'Taylor Swift – City of Lover'
-const venueName = 'Stadion Narodowy'
-const venueAddress = 'al. Księcia Józefa Poniatowskiego 1, 03-901 Warszawa'
-const isPaid = true
+const props = defineProps<{ eventId: number }>()
+
+const event = ref<RawEvent | null>(null)
+const loading = ref(true)
+
+onMounted(async () => {
+  try {
+    const response = await api.get<{ data: RawEvent }>(`/events/${props.eventId}`)
+    event.value = response.data.data
+  } catch (error) {
+    console.error('Błąd pobierania wydarzenia:', error)
+  } finally {
+    loading.value = false
+  }
+})
 </script>
 
 <template>
-  <AppHead title="Event" />
-  <div class="w-full mb-16 sm:mb-12 flex-col">
+  <AppHead :title="event?.title || 'Wydarzenie'" />
+  <div v-if="!loading && event" class="w-full mb-16 sm:mb-12 flex-col">
     <Navbar class="mb-[72px]" />
 
-    <div class="w-full h-[400px] bg-red-200 relative">
-      <img :src="bannerSrc" alt="Event Banner" class="size-full object-cover">
+    <div class="w-full h-[400px] relative bg-gray-200">
+      <img :src="event.image_url ?? 'https://picsum.photos/640/480'" alt="Event Banner" class="size-full object-cover">
 
       <div
         class="sm:hidden absolute left-1/2 -translate-x-1/2 bottom-[-32px] flex justify-between items-center bg-white rounded-full shadow px-6 py-3 w-fit max-w-full"
       >
         <p class="text-brand-dark font-medium whitespace-nowrap">
-          12k osób weźmie udział
+          {{ event.participants?.length || 0 }} osób weźmie udział
         </p>
-        <base-button
-          class="ml-4 h-[28px] bg-brand-dark text-white px-4 py-1 rounded-lg text-sm whitespace-nowrap"
-        >
+        <BaseButton class="ml-4 h-[28px] bg-brand-dark text-white px-4 py-1 rounded-lg text-sm whitespace-nowrap">
```

Comment:
Please use tailwind docs. https://v3.tailwindcss.com/docs/height

You can also migrate Tailwind from 3.x.x to the new 4.x.x standard, but of course in a separate PR.

```suggestion
        <BaseButton class="ml-4 h-7 bg-brand-dark text-white px-4 py-1 rounded-lg text-sm whitespace-nowrap">
```